### DISCUSSION TITLE
Welcome to gitpay Discussions!

### DISCUSSION THREAD

**Discussion by alexanmtz** [worknenjoy/gitpay#748] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hi everyone.

**Comment by alexanmtz** [comment]
Hello @codehangen, can you develop your idea better? It isn't apparent to me. Could you let me know if you sent the proper mockup? And your comment was cleared, but the one I've received doesn't seem right.

**Reply by ghost** [reply]
Hi alexanmtz.

> Hello @codehangen, can you develop your idea better? It isn't apparent to me. 

yes.

> Could you let me know if you sent the proper mockup? 

- I would like to create the best git client ever. 
- In that sense, I'm thinking of creating a git client based on the git client called: "guitar". I really like gitpay's proposal - which is to pay per bug. 
- An interesting use case I'm proposing is having a git client with gitpay. That way people can get paid directly in the git client.

> And your comment was cleared, but the one I've received doesn't seem right.

okay.

**Comment by alexanmtz** [comment]
Hey, @codehangen, I can see now your idea; yes, we already have some ideas, like a client to make the payment process more developer friendly, as studying to use our crypto-coin, and for now, we're improving our structure to be able to grow in that direction. We can develop this idea, and any idea will be very welcome!

**Reply by ghost** [reply]
Hi alexanmtz.

> Hey, @codehangen, I can see now your idea; yes, we already have some ideas, like a client to make the payment process more developer friendly, as studying to use our crypto-coin, and for now, we're improving our structure to be able to grow in that direction. 

thank you for feedback here. so that's exactly what I was thinking.

> We can develop this idea, and any idea will be very welcome!

Are you online? could you comment on this link: `https://github.com/worknenjoy/gitpay/discussions/943`?

**Comment by Digital365Staking** [comment]
Hi the team of Gitpay,
I would have a question about the project Gitpay : Can this project permit to register/connect every issue on GitHub with Gitpay ?
It seems there are not a lot of issues on the database of GitPay. I think it could be more fulfilled. The purpose of my idea is to grow the activity on the platform and establish a strong link with the whole github activity. Can it be easy to do ?

**Reply by Digital365Staking** [reply]
On this theme, an other question would be : how to attract a lot of companies on GitPay in the purpose they use more it and not only for issues but also for projects, ... ?

**Reply by alexanmtz** [reply]
Hey @Digital365Staking , thanks for your feedback.

We tried that in the past with Github Apps, but our Github app project is outdated:
https://github.com/worknenjoy/gitpay-github-app

But we will give this option in future versions to fully manage issues on Gitpay, but for now our next release will focus on improve our current features and user experience before we can grow the platform with more projects.

There's some challenges in fully connect with Github and along permissions and sync data, so we need to have a foundation to provide that and move to a way to import all issues from project, as new one created, as projects and organizations.

**Reply by Digital365Staking** [reply]
Hi Alex, I would like eventually to participate to this project. What kind of skills this project need ?

**Reply by alexanmtz** [reply]
It would be awesome @Digital365Staking , we are full JS stack with Postgres database, so we use React and Node.js

What skills do you have and like most?
